engine/src/main/java/org/entando/entando/web/page/model/PageRequest.java

```diff
@@ -30,7 +30,7 @@ public class PageRequest {
 
     @Size(max = 30, message = "string.size.invalid")
     @NotNull(message = "page.code.notBlank")
-    @Pattern(regexp = "[a-zA-Z0-9_-]+", message="page.code.wrongCharacters")
+    @Pattern(regexp = "[a-zA-Z0-9-_]+", message="page.code.wrongCharacters")

```

Comment:
is it possible to use a constant? the previous file has the same regexp ...

Reply:
Done